server/lib/reverse_etl/extractors/incremental_delta.rb

```diff
@@ -17,11 +17,14 @@ def read(sync_run_id, activity)
         batch_query_params = batch_params(source_client, sync_run)
         model = sync_run.sync.model
 
-        ReverseEtl::Utils::BatchQuery.execute_in_batches(batch_query_params) do |records, current_offset|
+        ReverseEtl::Utils::BatchQuery.execute_in_batches(batch_query_params) do |records,
+          current_offset, last_cursor_field_value|
```

Comment:
As of now current_offset in the cursor field not using.
 If I pass both current_offset and last_cursor_field_value in the same field, it will be confusing to update in the database because current_offset use in sync run, last_cursor_field_value  insync."